tests/frontend/use-user.test.tsx

```diff
@@ -157,4 +157,20 @@ describe('context wrapper', () => {
 
     expect(result.current.user).toEqual({ foo: 'bar' });
   });
+
+  test('should use the override fetch behaviour', async () => {
+    const fetchSpy = jest.fn();
+    (global as any).fetch = fetchSpy;
+
+    const customFetcher = jest.fn().mockReturnValue(Promise.resolve());
```

Comment:
```js
const customFetcher = jest.fn().mockReturnValue(Promise.resolve('foo'));
// then    
expect(result.current.user).toBe('foo');
```

Reply:
Ah, good shout. I've made that change.

Reply:
Looks good, thanks - I just need to wait a day or two for something before I can merge. Will keep you posted